sfge/src/main/java/com/salesforce/graph/symbols/apex/schema/SObjectFieldFactory.java

```diff
@@ -12,36 +14,51 @@
 @SuppressWarnings(
         "PMD.RedundantFieldInitializer") // SCHEMA_INDEX is intentionally zero to provide context
 public final class SObjectFieldFactory {
```

Comment:
Not bringing the chained names handling to fields at the moment. This brings in a lot more cases that need a lot of testing effort. Will revisit at the next opportunity.